.github/workflows/publish.yml

```diff
@@ -176,18 +175,36 @@ jobs:
         highest_semver_tag: ${{ needs.get-version.outputs.highest_semver_tag }}
 
   publish-python-sdk:
-    if: github.repository == 'feast-dev/feast'
     runs-on: ubuntu-latest
     needs: [build_wheels]
     steps:
+      - uses: actions/setup-python@v5
+        with:
+          python-version: '3.12'
+      - name: Install dependencies
+        run: |
+          python -m pip install --upgrade pip
+          pip install setuptools wheel twine
+      - uses: aws-actions/setup-sam@v2
+      - uses: aws-actions/configure-aws-credentials@v4
+        with:
+          aws-access-key-id: ${{ secrets.AWS_ACCESS_KEY_ID }}
+          aws-secret-access-key: ${{ secrets.AWS_SECRET_ACCESS_KEY }}
+          aws-region: eu-west-1
       - uses: actions/download-artifact@v4.1.7
         with:
           name: wheels
           path: dist
-      - uses: pypa/gh-action-pypi-publish@v1.4.2
-        with:
-          user: __token__
-          password: ${{ secrets.PYPI_PASSWORD }}
+      - name: Publish to CodeArtifact
+        env:
+          CODEARTIFACT_DOMAIN: zing-python
+          CODEARTIFACT_DOMAIN_OWNER: 127385700001
+          CODEARTIFACT_REPOSITORY: zing-python
+        run: |
+          export TWINE_USERNAME=aws
+          export TWINE_PASSWORD=$(aws codeartifact get-authorization-token --domain $CODEARTIFACT_DOMAIN --domain-owner $CODEARTIFACT_DOMAIN_OWNER --query authorizationToken --output text)
+          export TWINE_REPOSITORY_URL=$(aws codeartifact get-repository-endpoint --domain $CODEARTIFACT_DOMAIN --domain-owner $CODEARTIFACT_DOMAIN_OWNER --repository $CODEARTIFACT_REPOSITORY --format pypi --query repositoryEndpoint --output text)
+          python -m twine upload dist/*
 
   publish-java-sdk:
```

Comment:
I think this job should be deleted or disabled. It will fail every workflow run I think

Reply:
There should not be any problems with this, as we will not run this step in our repository due to the if-statement below.

We could delete this and other unused steps/actions altogether. But I suggest leaving them as they are, since the original repository is actively developing and I want to remove committees from it with a minimum number of conflicts